src/java.desktop/share/classes/javax/swing/border/AbstractBorder.java

```diff
@@ -181,6 +181,7 @@ public int getBaseline(Component c, int width, int height) {
      * @param c <code>Component</code> to return baseline resize behavior for
      * @return an enum indicating how the baseline changes as the border is
      *         resized
+     * @throws NullPointerException if passed in {@code Component} is {@code null}
```

Comment:
`if passed in {@code Component} is {@code null}` - Rephrasing this sentence would be better. Something like `if {@code Component} is {@code null}`.

Reply:
Updated in method description.